python/serialbox/serialization.py

```diff
@@ -432,10 +447,11 @@ def fieldnames(self):
         n_fields = fs_fields(self.serializer)
         name_lengths = (ctypes.c_int*n_fields)()
         fs_fieldname_lengths(self.serializer, name_lengths)
+        name_length = max(name_lengths)
 
         # Get field names
         names = ((ctypes.c_char_p)*n_fields)()
-        names[:] = [('\0'*name_lengths[i]).encode() for i in range(n_fields)]
+        names[:] = [('\0'*name_length).encode() for i in range(n_fields)]
```

Comment:
This is a good point. I think python has some magics to understand that the string is over even if there is no termination, but just to be in the safe side I will add one more character.